packages/atomic/src/components/atomic-search-interface/assets/dev.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Where would we put the "shared" strings for all components ?

For example, the keyword `Search` might be used by the searchbox, by facets, by query summary etc.

How could we share it between all those components ?

Reply:
To me, these are the only strings files we would need, I've thought of trying to move them back up the hierarchy if the stencil component assets allow it.

One json per language, should be light enough, one per component might be too overkill imo. 

A bit painful for dev to modify directly, adding a key in each file, we can leverage the build process, like Sami suggests, and have a json file with a structure similar to the one in the JSUI. Adding it to the todos.